waspc/packages/deploy/src/providers/fly/index.ts

```diff
@@ -0,0 +1,116 @@
+import { Command, Option } from 'commander';
+import { setup as setupFn } from './setup/setup.js';
+import { deploy as deployFn } from './deploy/deploy.js';
+import { createDb as createDbFn } from './createDb/createDb.js';
+import { cmd as cmdFn } from './cmd/cmd.js';
+import { launch as launchFn } from './launch/launch.js';
+import { ensureWaspDirLooksRight, ensureDirsInCmdAreAbsolute } from './helpers/helpers.js';
+import { ensureFlyReady, ensureRegionIsValid } from './helpers/flyctlHelpers.js';
+import { ContextOption } from './helpers/CommonOps.js';
+
+// Helpers for adding shared arguments and options.
+declare module 'commander' {
```

Comment:
Mainly used TS declaration merging so I could add a few helper methods to the class for the ergonomics of chained method calls. If this is an abuse, lemme know. 😅

Reply:
Haha, it's cool, but yeah... a bit abusive :smile: 

The "correct" way would be extending the class.